currency/manager.go

```diff
@@ -289,11 +289,12 @@ func (p *PairsManager) DisablePair(a asset.Item, pair Pair) error {
 		return err
 	}
 
-	enabled, err := pairStore.Enabled.Remove(pair)
-	if err != nil {
-		return err
+	if !pairStore.Enabled.Contains(pair, true) {
```

Comment:
Suggestion: If you want to minimise walk
```go
enabledLen := len(pairStore.Enabled)
	pairStore.Enabled = pairStore.Enabled.Remove(pair)

	if enabledLen == len(pairStore.Enabled) {
		return fmt.Errorf("%w %s", ErrPairNotFound, pair)
	}
```

Reply:
Huh. Actually fails the test 😮 
So, that's interesting.

Reply:
Fixed 132650e